inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -364,10 +374,10 @@ bool resolve_dynamic_shapes(const ngraph::Function& f) {
     return true;
 }
 
-void ngfunction_2_irv10(
-    pugi::xml_document& doc, std::vector<uint8_t>& bin,
-    ngraph::Function& f,
-    const std::map<std::string, ngraph::OpSet>& custom_opsets) {
+void ngfunction_2_irv10(pugi::xml_document& doc,
+                        std::ostream& bin_file,
+                        ngraph::Function& f,
```

Comment:
Can we use `const ngraph::Function`? If I understand right this method shouldn't change the nGraph Function.